Doc/library/wave.rst

```diff
@@ -47,6 +47,9 @@ The :mod:`wave` module defines the following function and exception:
    .. versionchanged:: 3.4
       Added support for unseekable files.
 
+   .. versionchanged:: 3.7
+      Added support for :term:`path-like object`.
```

Comment:
Or `:term:'path-like objects <path-like object>'` (use backquotes instead of apostrophes).